pymysqlreplication/event.py

```diff
@@ -7,11 +7,19 @@
 
 
 class BinLogEvent(object):
-    def __init__(self, from_packet, event_size, table_map, ctl_connection,
-                 only_tables = None,
-                 only_schemas = None,
-                 freeze_schema = False,
-                 fail_on_table_metadata_unavailable = False):
+    def __init__(
+        self,
+        from_packet,
+        event_size,
+        table_map,
+        ctl_connection,
+        only_tables=None,
+        ignored_tables=None,
+        only_schemas=None,
+        ignored_schemas=None,
+        freeze_schema=False,
+        fail_on_table_metadata_unavailable=False
+    ):
```

Comment:
I'm sorry to be picky but iI kind of dislike this style, could you please avoid making the diff too large and stick to the old style (which looks pep8 compliant besides the space around the operators)?  Same comment for `binlogstream.py` and `packet.py`.